### ISSUE TITLE
Initial Code Review

### ISSUE THREAD

**Issue by katemenkhaus** [PredixDev/px-panel#1]
- [x]  should hide the `x` icon when `persistent` is true in the demo, otherwise it seems broken
- [x]  it is nice to have a simple/example demo for testing and debugging 
- [x] the codepen appears broken
- [x] broken in shadow dom

**Comment by randyaskin** [comment]
- Added a dom-if in the demo for persistent.
- Added simple-demo.html. 
- Wade and I discussed the CodePen and decided it was fine if the demo displayed example content whereas the CodePen was simpler (also `localDomContent` wasn't working as expected for me).
- Switched to classes, which fixes the shadow dom issue.

**Comment by katemenkhaus** [comment]
- If you have`floating` true and `fixed` false the panel expands full with in its container. 

- the `open` and `close` buttons on the demo don't work in shadow dom 

-  _getClasses() could be more clear if it was something like _getContainerClasses

- Have we thought about keyboard short cuts for the open and close?

- When the panel is fixed but not floating it should not overlay the content. (Wade confirmed this)

- It might be worth taking another look at the `_onResize()` function. There is similar functionality in app-nav https://github.com/PredixDev/px-app-nav/blob/master/px-app-nav.es6.js#L416 and https://github.com/PredixDev/px-app-nav/blob/master/px-app-nav.es6.js#L396. We should make sure this is a performant as it can be. 

- If you checkout out the css-update branch I've reorganized the css. We can go over why I made the changes.
